client/my-sites/stats/activity-log-tasklist/index.jsx

```diff
@@ -445,55 +497,69 @@ const updateTheme = ( siteId, themeId ) =>
 		}
 	);
 
-const mapStateToProps = ( state, { siteId, plugins, themes } ) => {
+/**
+ * Start updating WordPress core on the specified site.
+ *
+ * @param {number} siteId  Site Id.
+ *
+ * @return {*} Stored data container for request.
+ */
+const updateCore = siteId =>
+	requestHttpData(
+		`core-update-${ siteId }`,
+		http( {
+			method: 'POST',
+			path: `/sites/${ siteId }/core/update`,
+			// No need to pass version: if it's missing, WP will be updated to latest core version.
+		} ),
+		{
+			fromApi: () => ( { version } ) => {
+				return [ [ version, true ] ];
+			},
+			freshness: -Infinity,
```

Comment:
`-Infinity` doesn't make much sense here. did you want to "force" this API call to happen? then `freshness: 0` is what we want. nothing can be less than `-Infinity` seconds old 😄 :